tests/encrypted_lvm.spec.ts

```diff
@@ -14,37 +15,69 @@ test.describe('The main page', () => {
     });
 
     test('Installation test with encrypted lvm file system', async ({ page }) => {
+        // optional actions done on the page
+        const actions = Object.freeze({
+          setProduct: Symbol("product"),
+          setMainPage: Symbol("main"),
+          done: Symbol("done")
+        });
+
+        // check for multiple texts in parallel, avoid waiting for timeouts
+        let action = await Promise.any([
+          page.getByText("Product selection").waitFor().then(() => actions.setProduct),
+          page.getByText(process.env.PRODUCTNAME).waitFor().then(() => actions.setMainPage),
+        ]);
+        
+        // optional product selection
+        if (action === actions.setProduct) {
+          await test.step("Select the product", async () => {
+            const productSelectionPage = new ProductSelectionPage(page);
+            await productSelectionPage.expectProductSelection();
+            await productSelectionPage.checkServerType();
+            await productSelectionPage.select();
+          });
+
+          action = await Promise.any([
```

Comment:
we don't need the second Promise.any, the root authentication is always not set.

Reply:
Updated